Benchmark/cmake/modules/Benchmark.cmake

```diff
@@ -41,9 +41,16 @@ function(add_benchmark CHECKID)
   target_link_libraries(${CHECKID}
     PUBLIC
       benchmark::benchmark
-      m
   )
 
+  find_library(MATH_LIBRARY m)
+  if(MATH_LIBRARY)
+    target_link_libraries(${CHECKID}
+      PUBLIC
+        ${MATH_LIBRARY}
+    )
+  endif()
```

Comment:
Thank you and my bad! ❤️

Reply:
Np, I didn't realize the issue either :grin: